mempool/mempool
# Lightning

### Testing

* Make sure you `state.schema_version` is set to 22 (current master)
* Make sure to get rid of all LN related tables in your db (`channels`, `lightning_stats`, `nodes`, `node_stats`)
* Make sure to delete `state.last_node_stats` entry
* Run the backend and pray
